WebApp/autoreduce_webapp/static/javascript/runs_list.js

```diff
@@ -8,22 +8,26 @@
         $('body').css('cursor', '');
     }
 
-    var init = function init(){
-        document.addEventListener("DOMContentLoaded", function(){
+    let init = function init() {
+        document.addEventListener("DOMContentLoaded", function () {
             setDefaultCursor($(this));
         });
 
-        var select = document.getElementById('filter_select');
-        select.onchange = function(){
-            setProgressCursor($(this));
-            document.getElementById('filter_options').submit();
-        };
+        let select = document.getElementById('filter_select');
+        if (select !== null) {
+            select.onchange = function () {
+                setProgressCursor($(this));
+                document.getElementById('filter_options');
+            };
+        }
 
-        var apply = document.getElementById('apply_filters');
-        apply.onclick = function(){
-            setProgressCursor($(this));
+        let apply = document.getElementById('apply_filters');
+        if (apply !== null) {
+            apply.onclick = function () {
+                setProgressCursor($(this));
+            };
         }
-    };
+    }
```

Comment:
I think the semicolon needs to stay because the start of this statement is `let int = ...`

Reply:
My mistake